crates/gas/src/event.rs

```diff
@@ -17,3 +17,17 @@ impl EventAttributeEntry<'static> for GasUsed {
         self.0
     }
 }
+
+/// Extend a [`namada_events::Event`] with the gas scale data.
+pub struct GasScale(pub u64);
```

Comment:
This seems to not be used

Reply:
done in 037f62f73b6c7fd1e9a73b8d485a67caa8ec0767